include/emp/meta/meta.hpp

```diff
@@ -122,19 +124,22 @@ namespace emp {
   //
   // Two helper functions exist to test each part: test_type_exist and test_type_value.
 
+  #ifndef DOXYGEN_SHOULD_SKIP_THIS
   namespace internal {
     template <template <typename...> class FILTER, typename T>
     constexpr bool tt_exist_impl(bool_decoy<FILTER<T>> x) { return true; }
     template <template <typename...> class FILTER, typename T>
     constexpr bool tt_exist_impl(...) { return false; }
   }
+  #endif DOXYGEN_SHOULD_SKIP_THIS
```

Comment:
should this be `#endif // DOXYGEN_SHOULD_SKIP_THIS`

Reply:
looks like there are several other instances of this, if needs to be fixed do a quick project search to make sure you get 'em all

Reply:
Oh shoot, it's a little concerning that the tests didn't catch that

Reply:
Okay, think I got them all